# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xawtv/quicktime-headers.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN xawtv-3.95.orig/libng/plugins/read-qt.c xawtv-3.95/libng/plugins/read-qt.c
--- xawtv-3.95.orig/libng/plugins/read-qt.c	2003-02-14 15:14:05.000000000 +0100
+++ xawtv-3.95/libng/plugins/read-qt.c	2007-10-07 16:34:17.000000000 +0200
@@ -4,9 +4,9 @@
 #include <stdio.h>
 #include <string.h>
 #include <pthread.h>
-#include <quicktime/quicktime.h>
-#include <quicktime/colormodels.h>
-#include <quicktime/lqt.h>
+#include <lqt/quicktime.h>
+#include <lqt/colormodels.h>
+#include <lqt/lqt.h>
 
 #include "grab-ng.h"
 
diff -ruN xawtv-3.95.orig/libng/plugins/write-qt.c xawtv-3.95/libng/plugins/write-qt.c
--- xawtv-3.95.orig/libng/plugins/write-qt.c	2003-02-14 15:14:06.000000000 +0100
+++ xawtv-3.95/libng/plugins/write-qt.c	2007-10-07 16:32:27.000000000 +0200
@@ -4,9 +4,9 @@
 #include <stdio.h>
 #include <string.h>
 #include <pthread.h>
-#include <quicktime/quicktime.h>
-#include <quicktime/colormodels.h>
-#include <quicktime/lqt.h>
+#include <lqt/quicktime.h>
+#include <lqt/colormodels.h>
+#include <lqt/lqt.h>
 
 #include "grab-ng.h"
 
